pvlib/modelchain.py

```diff
@@ -1798,6 +1842,50 @@ def run_model_from_effective_irradiance(self, data):
             of Arrays in the PVSystem.
         ValueError
             If the DataFrames in `data` have different indexes.
+         Examples
```

Comment:
```suggestion

        Examples
```

I think this will fix the formatting for `run_model_from_effective_irradiance`